frontend/js/pages/CardTeam.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
```

Comment:
What do you think of putting this on the components folder?

Reply:
thanks :)